package.json

```diff
@@ -73,20 +73,24 @@
   },
   "dependencies": {
     "@bigbinary/neeto-icons": "^1.4.0",
+    "@popperjs/core": "^2.9.2",
```

Comment:
@amaldinesh7 Are we using this package. I don't find an import. Can you remove it?

Reply:
It is a dependency package of react-popper. @edwinbbu